mta-sts.md

```diff
@@ -439,6 +432,22 @@ An example control flow for a compliant sender consists of the following steps:
    the case of temporary message delivery failures apply (as discussed in
    [@!RFC5321] section 4.5.4.1).
 
+# Reporting Failures
+
+MTA-STS is intended to be used along with TLSRPT (TODO: add ref) in order to
+ensure implementing domains can detect cases of both benign and malicious
+failures, and to ensure that failures that indicate an active attack are
+discoverable. As such, senders who also implement TLSRPT SHOULD treat the
+following events as reportable failures:
+
+* HTTPS policy fetch failures when a valid TXT record is present.
```

Comment:
If the TXT record is present, but the policy is not present (say the admin didn't host a HTTPS endpoint), what is the behavior from the sender side? Are they continue to fetch policy based on the TXT record DNS TTL? 

It may lead to DOS attacks. So my questions:
1. Do senders need to follow exponential backoff?
2. Is it possible to have MIN and MAX for DNS TTL?

Reply:
It's a good question. Regarding your first point, we suggested as follows

>    Senders may wish to rate-limit the frequency of attempts to fetch the
>    HTTPS endpoint even if a valid TXT record for the recipient domain
>    exists.  In the case that the HTTPS GET fails, we suggest
>    implementions may limit further attempts to a period of five minutes
>    or longer per version ID, to avoid overwhelming resource-constrained
>    recipients with cascading failures.

Regarding #2, no. What we could do is say that an HTTP GET failure should be cached (in some manner) until the DNS TTL expires, but I think it's OK as-is, where we make a suggestion but othewrise leave it to implementations to do the right thing. 5 minutes, exponential backoff--both are quite reasonable; in practice it probably doesn't matter a huge amount. 

WDYT?